test/unit/legacy_wrappers/db.test.js

```diff
@@ -40,151 +41,8 @@ describe('legacy_wrappers/db.js', () => {
-        });
-      });
-    });
-
-    describe(`and addUser is called with ('name', options, callback)`, () => {
-      beforeEach(() => {
-        actualReturnValue = db.addUser('name', { options: true }, callback);
-      });
-
-      it('should return void', () => expect(actualReturnValue).to.be.undefined);
-
-      it('should call the callback with undefined error and successful result', async () => {
-        await runMicroTask();
-        expect(callback).to.have.been.calledOnceWith(undefined, { message: 'success!' });
-      });
-
-      it(`should pass only ('name', undefined, options) to the driver api`, () => {
-        expect(stubbedMethod).to.have.been.calledOnceWithExactly('name', undefined, {
-          options: true
-        });
-      });
-    });
-
-    describe(`and addUser is called with ('name')`, () => {
-      beforeEach(() => {
-        actualReturnValue = db.addUser('name');
-      });
-
-      it('should return the same promise the driver returns', async () => {
-        expect(actualReturnValue).to.equal(superPromise);
-      });
-
-      it('should return a resolved promise', async () => {
-        const result = await actualReturnValue;
-        expect(result).to.have.property('message', 'success!');
-      });
-
-      it(`should pass only ('name') to the driver api`, () => {
-        expect(stubbedMethod).to.have.been.calledOnceWithExactly('name', undefined, undefined);
-      });
-    });
-
-    describe(`and addUser is called with ('name', 'pass')`, () => {
-      beforeEach(() => {
-        actualReturnValue = db.addUser('name', 'pass');
-      });
-
-      it('should return the same promise the driver returns', async () => {
-        expect(actualReturnValue).to.equal(superPromise);
-      });
-
-      it('should return a resolved promise', async () => {
-        const result = await actualReturnValue;
-        expect(result).to.have.property('message', 'success!');
-      });
-
-      it(`should pass only ('name', 'pass') to the driver api`, () => {
-        expect(stubbedMethod).to.have.been.calledOnceWithExactly('name', 'pass', undefined);
-      });
-    });
-
-    describe(`and addUser is called with ('name', 'pass', options)`, () => {
-      beforeEach(() => {
-        actualReturnValue = db.addUser('name', 'pass', { options: true });
-      });
-
-      it('should return the same promise the driver returns', async () => {
-        expect(actualReturnValue).to.equal(superPromise);
-      });
-
-      it('should return a resolved promise', async () => {
-        const result = await actualReturnValue;
-        expect(result).to.have.property('message', 'success!');
-      });
-
-      it(`should pass only ('name', 'pass', options) to the driver api`, () => {
-        expect(stubbedMethod).to.have.been.calledOnceWithExactly('name', 'pass', {
-          options: true
-        });
-      });
-    });
+  it('should return legacy Admin', () => {
+    expect(db.admin()).to.be.instanceOf(LegacyAdmin);
```

Comment:
Was this migrated from `test/unit/legacy_wrappers/admin.test.js`?

Reply:
This was to achieve 100% coverage again 😅 After removing the `addUser` API, nothing called `db.admin()` in these tests anymore but we should've had this covered specifically originally, it just was implicitly covered because addUser wouldn't have worked with callbacks if not.